### PR TITLE
waypoint on-demand runner inspect and small copy changes

### PR DESCRIPTION
**Pull request by mitchellh**
PR into @evanphx's branch.

### CODE REVIEW

File: internal/cli/ondemand_runner_inspect.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	id := c.args[0]
+
+	resp, err := c.project.Client().GetOnDemandRunnerConfig(c.Ctx, &pb.GetOnDemandRunnerConfigRequest{
+		Config: &pb.Ref_OnDemandRunnerConfig{Id: id},
+	})
+	if err != nil {
+		c.ui.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
+		return 1
+	}
+	fmt.Println(proto.MarshalTextString(resp.Config))
```
Comment (briancain) [inline internal/cli/ondemand_runner_inspect.go:43]: We've been trying to keep `inspect` commands looking the same, so I think this should output with the UI.NamedVars and a header before the config prints

**Review by briancain — CHANGES_REQUESTED** [review]
Minor comment but looks good to me otherwise!

File: internal/cli/ondemand_runner_inspect.go (hunk 1 above)
Comment (mitchellh) [reply internal/cli/ondemand_runner_inspect.go:43]: Can you show an example? I copied this from `auth-method inspect` haha. I'll copy whatever style.

Comment (briancain) [reply internal/cli/ondemand_runner_inspect.go:43]: Ah, definitely!

I think initially we did it on `waypoint context inspect` and then it was suggested I keep that with https://github.com/hashicorp/waypoint/pull/2080.

Comment (mitchellh) [reply internal/cli/ondemand_runner_inspect.go]: CleanShot 2021-08-19 at 16 58 15@2x

Done!

**Review by briancain — APPROVED** [review]
Awesome! 👍🏻 ✨

**Pull request merged**